simulation/sub8_montecarlo/nodes/example_verification.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+'''Monte-Carlo controller verification
```

Comment:
Removed the extraneous test for this, and changed example_verification into a plotting tool